.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "gotoSymbolStack.currentStackPosition": 0,
+    "gotoSymbolStack.maxStackPosition": 0,
+    "gotoSymbolStack.filePositionInfo": []
+}
```

Comment:
I don't know the background about these configuration. Could you explain what these done? Thanks!